playwright.config.ts

```diff
@@ -39,31 +39,31 @@ const config: PlaywrightTestConfig = {
     // baseURL: 'http://localhost:3000',
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
```

Comment:
It makes sense to keep a trailing comma, so the same line won't be added to the next commit if more items/props are added.

Reply:
True, it's cleaner that way.